package.json

```diff
@@ -100,23 +101,23 @@
     "icon": "https://user-images.githubusercontent.com/321738/63501763-88dbf600-c4cc-11e9-96cd-94adadc2fd72.png"
   },
   "dependencies": {
-    "@babel/core": "^7.18.13",
-    "@babel/generator": "^7.18.13",
-    "@babel/preset-env": "^7.19.4",
+    "@babel/core": "^7.20.7",
+    "@babel/generator": "^7.20.7",
+    "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
-    "@babel/template": "^7.14.5",
-    "@babel/types": "^7.14.8",
+    "@babel/template": "^7.20.7",
+    "@babel/types": "^7.20.7",
     "@storybook/core-common": "next",
     "@storybook/csf": "next",
     "@storybook/csf-tools": "next",
-    "@storybook/store": "next",
+    "@storybook/preview-api": "next",
```

Comment:
Not sure if this should be done in this PR. It will be breaking for any user with sb < alpha 52, and some people can't upgrade as they're blocked on storystorev6 issues.

Reply:
Ah, okay, I got some deprecation warnings in my console, about using `@storybook/store`, but I will revert then.